src/renderer/components/shared/bsm-button.component.tsx

```diff
@@ -84,18 +93,35 @@ export const BsmButton = forwardRef<unknown, Props>(({ className, style, iconSty
 
     const handleClick = (e: MouseEvent<HTMLDivElement>) => !disabled && onClick?.(e);
 
+    const renderTooltip = () => {
+        return (
+            <Tippy
+                className="!bg-main-color-1"
+                content={t(tooltip)}
```

Comment:
I meant, adding a tooltip while hovering the button globally, like this users are sure to see the tooltip.
So no need to add all this stuff in the button component

Reply:
Ahhh, sorry about that, thought we needed an icon for it, will apply a commit for this.